io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/InstanceExportConfigurationPage.java

```diff
@@ -103,6 +104,8 @@ public void disable() {
 	public boolean updateConfiguration(InstanceWriter provider) {
 		provider.setParameter(InstanceTableIOConstants.SOLVE_NESTED_PROPERTIES,
```

Comment:
I also don't know what this parameter is for and I have seen also comments in the code asking the same question (`CSVInstanceWriter.java` lines 79-81)

However, how do you deduce that this is not used in case useSchema=true?

Reply:
I think I was wrong and it can apply even with `useSchema`. Though I'm not sure what the result is because I did not analyse the `iterateBuild` function.